src/main/java/com/ericsson/ei/notifications/InformSubscriber.java

```diff
@@ -181,16 +183,17 @@ private void saveMissedNotificationToDB(String missedNotification) {
      * @return String
      */
     private String prepareMissedNotification(String aggregatedObject, String subscriptionName,
-            String notificationMeta) {
+            String notificationMeta, String errorMessage) {
         BasicDBObject document = new BasicDBObject();
         document.put("subscriptionName", subscriptionName);
         document.put("notificationMeta", notificationMeta);
         try {
-            document.put("Time", DateUtils.getDate());
+            document.put("time", DateUtils.getDate());
         } catch (ParseException e) {
             LOGGER.error("Failed to get date object.", e);
         }
-        document.put("AggregatedObject", BasicDBObject.parse(aggregatedObject));
+        document.put("aggregatedObject", BasicDBObject.parse(aggregatedObject));
+        document.put("message", errorMessage);
```

Comment:
It would be useful to have an example of a failed notification in the documentation - showing the user that this object will saved in the database. Funny thing, we don't even explain in the user docs how EI handles failures when attempting to contact subscribers... 

The only hint we give is this: https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/step-by-step-subscription-notification.md#missed-notifications 
but I think it should perhaps be explained more. And updated with example reflecting the changes made in this PR.

Reply:
Sure, I can put some examples in the docs. Maybe mention the http thing in notification meta as well.